src/xdocs/config_coding.xml

```diff
@@ -1365,24 +1365,26 @@ public class MyClass {
           <tr>
             <td>setterCanReturnItsClass</td>
             <td>
+              Allow to ignore the parameter of a property setter
+              method returning class in which setter is defined.
               Used in conjunction with <code>ignoreSetter</code> property it
               controls rule that recognizes method as a setter. By default
               setter is a method with signature of type
-              <pre>
-                void setXyz(${someType} xyz)
-              </pre>
+              <source>
+void setXyz(${someType} xyz)
+              </source>
               By setting this property (<code>setterCanReturnItsClass</code>)
               to <code>true</code> we expand definition of setter to also
               include returning class in which setter is defined. For example
-              <pre>
-                class Foo {
-                    int prop;
-                    Foo setProp(int prop) {
-                        this.prop = prop;
-                        return this;
-                    }
-                }
-              </pre>
+              <source>
```

Comment:
Won't changing this to `source` inside the property table put a block box around it? Wouldn't that look a bit confusing?
I also question having such examples in property table and not being in the description in somewhere else.